### PR TITLE
[TRUS-2680]  - Add Personal Representative -Individual

### PR DESCRIPTION
**Pull request by jassalrichy**

### CODE REVIEW

File: conf/individual.routes [hunk 1]
```diff
@@ -25,15 +25,15 @@ POST       /passport-or-idcard                              controllers.individu
...
 
-GET        /passport                                        controllers.individual.PassportDetailsController.onPageLoad(mode: Mode = NormalMode)
-POST       /passport                                        controllers.individual.PassportDetailsController.onSubmit(mode: Mode = NormalMode)
-GET        /change-passport                                 controllers.individual.PassportDetailsController.onPageLoad(mode: Mode = CheckMode)
-POST       /change-passport                                 controllers.individual.PassportDetailsController.onSubmit(mode: Mode = CheckMode)
+GET        /passport                                        controllers.individual.add.PassportDetailsController.onPageLoad()
+POST       /passport                                        controllers.individual.add.PassportDetailsController.onSubmit()
+GET        /change-passport                                 controllers.individual.add.PassportDetailsController.onPageLoad()
+POST       /change-passport                                 controllers.individual.add.PassportDetailsController.onSubmit()
```
Comment (jackheslop96) [inline conf/individual.routes:31]: No longer needed

File: conf/individual.routes [hunk 2]
```diff
@@ -25,15 +25,15 @@ POST       /passport-or-idcard                              controllers.individu
...
 
-GET        /id-card                                         controllers.individual.IdCardDetailsController.onPageLoad(mode: Mode = NormalMode)
-POST       /id-card                                         controllers.individual.IdCardDetailsController.onSubmit(mode: Mode = NormalMode)
-GET        /change-id-card                                  controllers.individual.IdCardDetailsController.onPageLoad(mode: Mode = CheckMode)
-POST       /change-id-card                                  controllers.individual.IdCardDetailsController.onSubmit(mode: Mode = CheckMode)
+GET        /id-card                                         controllers.individual.add.IdCardDetailsController.onPageLoad()
+POST       /id-card                                         controllers.individual.add.IdCardDetailsController.onSubmit()
+GET        /change-id-card                                  controllers.individual.add.IdCardDetailsController.onPageLoad()
+POST       /change-id-card                                  controllers.individual.add.IdCardDetailsController.onSubmit()
```
Comment (jackheslop96) [inline conf/individual.routes:36]: No longer needed

File: conf/individual.routes (hunk 1 above)
Comment (jassalrichy) [reply conf/individual.routes]: Removed

File: conf/individual.routes (hunk 2 above)
Comment (jassalrichy) [reply conf/individual.routes]: Removed

File: app/controllers/individual/PassportOrIdCardController.scala [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      form.bindFromRequest().fold(
+        (formWithErrors: Form[_]) =>
+          Future.successful(BadRequest(view(formWithErrors, request.individualName, mode))),
+
+        value => {
+          for {
+            updatedAnswers <- Future.fromTry(request.userAnswers.set(PassportOrIdCardPage, value))
+            _              <- registrationsRepository.set(updatedAnswers)
+          } yield {
+            value match {
```
Comment (jaynabonne) [inline app/controllers/individual/PassportOrIdCardController.scala:68]: Is there any reason why this doesn't use the Navigator as others do?

**Pull request merged**